src/core/database/database.service.ts

```diff
@@ -156,6 +205,14 @@ export class DatabaseService {
     nodevar: string;
   }): Promise<TObject> {
     const createdAt = DateTime.local();
+    let nodePropsToUpdate: SelectedNodeProperties = { createdAt, value };
+    if (await this.hasSortValue({ session, object, key, nodevar })) {
```

Comment:
We should be able to use the same logic as we do on create to determine the sort value. We shouldn't need to pull the current value out of the database to determine this.